cerberus-org/cerberus-app
# Volunteer API

# #6 #11 

## Testing
1. Run with `npm start`
2. Use the postman collection to test CRUD functionality
3. Ensure the following:

- [x] Volunteers can be inserted

- [x] Get by ID and get all return the inserted volunteer(s)

- [x] Count all volunteers returns the correct value

- [x] Volunteers can be updated

- [x] All volunteers have unique first/last/pet name combinations, test using insert and update

- [x] Get/update/delete by nonexistent ID returns 404 NOT FOUND
